### PR TITLE
Adapt trollflow2 for removal of ppp_config_dir

### LINKED ISSUE #96: Latest trollflow2 master fails with latest satpy master
**Describe the bug**

The latest trollflow2 master `create_scene` plugin fails with the latest satpy master, due to satpy's removal of `ppp_config_dir` in https://github.com/pytroll/satpy/pull/1501, which causes trollflow2 to fail with `TypeError: __init__() got an unexpected keyword argument 'ppp_config_dir'`.

**To Reproduce**

```python
from trollflow2.plugins import create_scene
job = {"input_filenames": "bar", "product_list": {}}
create_scene(job)
```

**Expected behavior**

Should probably fail because the input filenames don't exist.

**Actual results**

Actually fails to create a `Scene` object in the first place.

```
Traceback (most recent call last):
  File "mwe003.py", line 3, in <module>
    create_scene(job)
  File "/data/gholl/miniconda3/envs/py38b/lib/python3.8/site-packages/trollflow2/plugins/__init__.py", line 78, in create_scene
    job['scene'] = Scene(filenames=job['input_filenames'], **conf)
TypeError: __init__() got an unexpected keyword argument 'ppp_config_dir'
```

Or in an actual test production environment:

```
[DEBUG: 2021-02-03 08:00:22 : Listener] New message received: pytroll://aapp/avhrr file pytroll@oflks333.dwd.de 2021-02-03T08:00:22.896285 v1.01 application/json {"path": "", "platform_name": "Metop-A", "start_time": "2021-02-03T07:43:18", "end_time": "2
021-02-03T07:52:12", "processing_time": "2021-02-03T07:43:19", "sensor": "avhrr/3", "orig_platform_name": "M02", "collection_area_id": "eurol", "filename": "hrpt_metop02_20210203_0743_74150.l1b", "uri": "file://localhost/data/pytroll/TMP/AAPP-processed/metop02_20210203_0743_74150/hrpt_metop02_20210203_0743_74150.l1b", "uid": "hrpt_metop02_20210203_0743_74150.l1b", "orbit_number": 74150, "format": "AAPP", "type": "Binary", "data_processing_level": "1B", "station": "unknown", "env": "test"}
[INFO: 2021-02-03 08:00:22 : trollflow2.launcher] Distributed processing not configured, using default scheduler
[INFO: 2021-02-03 08:00:22 : trollflow2.plugins] Generating scene
[ERROR: 2021-02-03 08:00:22 : trollflow2.launcher] Process crashed
Traceback (most recent call last):
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/site-packages/trollflow2/launcher.py", line 268, in process
    cwrk.pop('fun')(job, **cwrk)
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/site-packages/trollflow2/plugins/__init__.py", line 78, in create_scene
    job['scene'] = Scene(filenames=job['input_filenames'], **conf)
TypeError: __init__() got an unexpected keyword argument 'ppp_config_dir'
[DEBUG: 2021-02-03 08:00:22 : trollflow2.launcher] Cleaning up
Process Process-20:
Traceback (most recent call last):
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/site-packages/trollflow2/launcher.py", line 268, in process
    cwrk.pop('fun')(job, **cwrk)
  File "/opt/pytroll/pytroll_inst/miniconda3/envs/pytroll-py38/lib/python3.8/site-packages/trollflow2/plugins/__init__.py", line 78, in create_scene
    job['scene'] = Scene(filenames=job['input_filenames'], **conf)
TypeError: __init__() got an unexpected keyword argument 'ppp_config_dir'
[ERROR: 2021-02-03 08:00:22 : trollflow2.launcher] Process crashed with exit code 1
```
Text output of actual results or error messages including full tracebacks if applicable.

**Environment Info:**
 - OS: `centOS 7.6.1810`
 - Trollflow2 Version: `v0.8.0-5-g7acb09b`
 - Satpy Version: `v0.25.1-49-ge0f1fb83`

**Additional context**

This isn't caught by the `test_create_scene` unit test, because it mocks the `Scene` call so fails to notice this keyword argument is not valid.

### PR DESCRIPTION
**Pull request by gerritholl**
Adapt the trollflow2 `create_scene` plugin for the removal of
ppp_config_dir.  Fixes #96.

 - [x] Closes #96 
 - [x] Tests added  
 - [x] Tests passed 
 - [x] Passes ``flake8 trollflow2`` 
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Review by mraspaud — COMMENTED** [review]
LGTM.

File: AUTHORS.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ The following people have made contributions to this project:
 - [Adam Dybbroe (adybbroe)](https://github.com/adybbroe)
 - [Panu Lahtinen (pnuu)](https://github.com/pnuu)
 - [Martin Raspaud (mraspaud)](https://github.com/mraspaud)
+- [Gerrit Holl (gerritholl)](https://github.com/gerritholl)
```
Comment (mraspaud) [inline AUTHORS.md:12]:
```suggestion
- [Gerrit Holl (gerritholl)](https://github.com/gerritholl)
- [Panu Lahtinen (pnuu)](https://github.com/pnuu)
- [Martin Raspaud (mraspaud)](https://github.com/mraspaud)
```

**Review by pnuu — APPROVED** [review]
LGMT

File: trollflow2/tests/test_trollflow2.py [hunk 2]
```diff
@@ -531,18 +531,27 @@ class TestCreateScene(TestCase):
     def test_create_scene(self):
         """Test making a scene."""
         from trollflow2.plugins import create_scene
-        with mock.patch("trollflow2.plugins.Scene") as scene:
+        from satpy.version import version as satpy_version
```
Comment (pnuu) [inline trollflow2/tests/test_trollflow2.py:534]:
```suggestion
        try:
            from satpy.version import version as satpy_version
        except ImportError:
            satpy_version = "0.26.0"
```

File: trollflow2/plugins/__init__.py [hunk 3]
```diff
@@ -68,8 +68,10 @@ class AbortProcessing(Exception):
 def create_scene(job):
     """Create a satpy scene."""
     defaults = {'reader': None,
-                'reader_kwargs': None,
-                'ppp_config_dir': None}
+                'reader_kwargs': None}
+    from satpy.version import version as satpy_version
```
Comment (pnuu) [inline trollflow2/plugins/__init__.py:72]: This should be moved to module level and mocked in the tests so that we don't need to actually install Satpy.

**Review by pnuu — CHANGES_REQUESTED** [review]
Couple of suggested changes so that we don't need to install Satpy for CI, which would increase the build times by a more than ten minutes (IIRC).

File: trollflow2/plugins/__init__.py (hunk 3 above)
Comment (gerritholl) [reply trollflow2/plugins/__init__.py]: I moved it to the module level.

I guess the `try:...except:...` counts as a mock, or do you mean actually via the mock module?

**Review by pnuu — APPROVED** [review]
LGTM, thanks for the update!

**Pull request merged**